pkg/nsx/services/inventory/build.go

```diff
@@ -443,7 +455,7 @@ func (s *InventoryService) BuildNode(node *corev1.Node) (retry bool) {
 		ContainerClusterId: s.NSXConfig.Cluster,
 		ExternalId:         string(node.UID),
 		IpAddresses:        ipAddresses,
-		NetworkErrors:      nil,
+		NetworkErrors:      networkErrors,
 		NetworkStatus:      status,
```

Comment:
if the status is HEALTH, no need to report the NetworkErrors? 
for node, even it's ready, there still were lots of condition.Message

Reply:
fixed